api/pom.xml

```diff
@@ -93,11 +93,26 @@
         <dependency>
             <groupId>jakarta.el</groupId>
             <artifactId>jakarta.el-api</artifactId>
-            <version>5.0.0</version>
+            <version>6.0.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
     </dependencies>
 
+    <repositories>
+      <repository>
+        <name>Jakarta EE Snapshots</name>
+        <id>snapshots-repo</id>
+        <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots/</url>
```

Comment:
Isn't this part of the parent pom?

Reply:
Yes it is. My mistake for thinking the "effective POM" view in Eclipse would show me all the profiles. -Psnapshots does the trick. I'll drop this commit from the PR.